runtime/kusama/src/lib.rs

```diff
@@ -223,7 +217,8 @@ impl pallet_scheduler::Config for Runtime {
 	type PalletsOrigin = OriginCaller;
 	type RuntimeCall = RuntimeCall;
 	type MaximumWeight = MaximumSchedulerWeight;
-	type ScheduleOrigin = EitherOf<EnsureRoot<AccountId>, AuctionAdmin>;
+	// TODO: Think more carefully and possibly replace with lower privilege origin.
+	type ScheduleOrigin = EnsureRoot<AccountId>;
```

Comment:
Let's not merge with a TODO, but we can discuss here.

The goal of having `ScheduleOrigin` include `AuctionAdmin` is to allow the auctions track of OpenGov to schedule periodic auctions.

AFAIK the `Scheduler` pallet simply forwards calls with the runtime origin, so any origin that can schedule calls is still limited only to calls permissioned to that origin.

Is the concern here about filling up the scheduler with junk?

Removing `AuctionAdmin` would make every single auction from now on either a one-off `AuctionAdmin` referendum or scheduled on the `root` track.